Github dependency must be updated properly.

```
(master c9097cf) p@m /workspace/ckeditor5> npm run release:dependencies

> ckeditor5@1.0.0-beta.1 release:dependencies /workspace/ckeditor5
> node ./scripts/release/release-dependencies.js

/workspace/ckeditor5-dev/node_modules/github/index.js:1
(function (exports, require, module, __filename, __dirname) { throw new Error("'github' has been renamed to '@octokit/rest' (https://git.io/vNB11)")
                                                              ^

Error: 'github' has been renamed to '@octokit/rest' (https://git.io/vNB11)
    at Object.<anonymous> (/workspace/ckeditor5-dev/node_modules/github/index.js:1:69)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/workspace/ckeditor5-dev/packages/ckeditor5-dev-env/lib/release-tools/utils/creategithubrelease.js:8:19)
    at Module._compile (module.js:570:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ckeditor5@1.0.0-beta.1 release:dependencies: `node ./scripts/release/release-dependencies.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ckeditor5@1.0.0-beta.1 release:dependencies script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/p/.npm/_logs/2018-04-10T08_33_35_291Z-debug.log
(master c9097cf) p@m /workspace/ckeditor5>
```

--- a/packages/ckeditor5-dev-env/lib/release-tools/utils/creategithubrelease.js
+++ b/packages/ckeditor5-dev-env/lib/release-tools/utils/creategithubrelease.js
@@ -5,7 +5,7 @@
 
 'use strict';
 
-const GitHubApi = require( 'github' );
+const GitHubApi = require( '@octokit/rest' );
 
 /**
  * Create a Github release.

--- a/packages/ckeditor5-dev-env/package.json
+++ b/packages/ckeditor5-dev-env/package.json
@@ -6,6 +6,7 @@
   "main": "lib/index.js",
   "dependencies": {
     "@ckeditor/ckeditor5-dev-utils": "^8.0.0",
+    "@octokit/rest": "^14.0.0",
     "chalk": "^2.1.0",
     "compare-func": "^1.3.2",
     "concat-stream": "^1.6.0",
@@ -15,7 +16,6 @@
     "del": "^3.0.0",
     "fs-extra": "^5.0.0",
     "git-raw-commits": "^1.2.0",
-    "github": "^9.2.0",
     "glob": "^7.1.2",
     "inquirer": "^5.2.0",
     "minimatch": "^3.0.4",

--- a/packages/ckeditor5-dev-env/tests/release-tools/utils/creategithubrelease.js
+++ b/packages/ckeditor5-dev-env/tests/release-tools/utils/creategithubrelease.js
@@ -33,7 +33,7 @@ describe( 'dev-env/release-tools/tasks', () => {
 				warnOnUnregistered: false
 			} );
 
-			mockery.registerMock( 'github', function GitHubApi() {
+			mockery.registerMock( '@octokit/rest', function GitHubApi() {
 				return {
 					authenticate: stubs.authenticate,
 					repos: {

